codes/quantum/groups/g-covariant.yml

```diff
@@ -0,0 +1,49 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: g_covariant
+physical: groups
+logical: qudits
+
+name: '\(G\)-covariant code'
+introduced: 'doi:10.1103/PRXQuantum.2.010326'
+
+description: |
+  A general class of codes where the encoding map is equivariant (i.e. commutes) 
+  with group actions on the logical and physical spaces.  The physical space is in 
+  general comprised of \(n\) distinct subsystems, some possibly being infinite 
+  dimensional.  The physical group action is the unitary tensor product
+  representation, making physical group actions intrinsically transversal with 
+  respect to the \(n\)-subsystem decomposition.
+
+protection: |
+  The noise model is an erasure of a known subset of the physical subsystems.  
+  The ability to perfectly protect however depends on both the group and the 
+  physical space.  A partial classification result is the following: (need to 
+  figure out how to make yml bulletin).  A. The group contains a continuous

```

Comment:
Is it like markdown? i.e.
- first item
- second item
You can also preview your submission using the https://github.com/errorcorrectionzoo/eczoo_generator repo

Reply:
@phfaist may know a better way

Reply:
There's no support for lists or enumeration environments (yet). I believe we once discussed this @valbert4 but thought that it risked adding too much additional visual structure to the page layout (I'm not sure I recall exactly what we said). I think it would make sense to add support for such environments; what do you think?

Reply:
There's only been an isolated number of cases where this would have made life better.